server/datastore/mysql/mysql.go

```diff
@@ -326,6 +328,18 @@ func registerTLS(config config.MysqlConfig) error {
 // provided configuration.
 func generateMysqlConnectionString(conf config.MysqlConfig) string {
 	tz := url.QueryEscape("'-00:00'")
+
+	// Check if file exists on disk
+	// If file exists read contents
+	if _, err := os.Stat(conf.Password); err == nil {
+		fileContents, err := ioutil.ReadFile(conf.Password)
+		if err != nil {
+			panic(err)
```

Comment:
Let's move the file read out into the `New` function so that we can return an error rather than panicking.